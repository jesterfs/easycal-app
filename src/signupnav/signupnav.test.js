import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import SignupNav from './signupnav'
import renderer from 'react-test-renderer';

describe('SignupNav component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <SignupNav />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <SignupNav />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});