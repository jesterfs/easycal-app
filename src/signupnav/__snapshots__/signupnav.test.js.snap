// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupNav component renders the UI as expected 1`] = `
<div
  className="SignUpNav, group"
>
  <div
    className="item"
  />
  <div
    className="item"
  >
    <h1>
      EasyCal
    </h1>
  </div>
  <div
    className="item navbtndiv"
  >
    <a
      href="/login"
      onClick={[Function]}
    >
       
      <button
        className="navbtn"
      >
        Log In
      </button>
       
    </a>
  </div>
</div>
`;
